### ISSUE TITLE
Add cumulative frequency of attendanc by reputables

### ISSUE THREAD

**Issue by brenzi** [encointer/accounting-frontend#3]
Like fig 3 in https://accounting.encointer.org/rewards-report

**Comment by pifragile** [comment]
what is the x axis of this plot? and how do we exactly measure this? because our model is that people should use different keys for different cycles, right?

**Comment by pifragile** [comment]
@brenzi  this chart only works under the assumption that people always use the same account. this kinda contradicts our assumptions, right? should we still proceed with it?

**Comment by brenzi** [comment]
True, but thats the way people use the protocol, currently. So I think we can proceed anyway
